crowdsource/resolvers/person.py

```diff
@@ -0,0 +1,91 @@
+from opencivicdata.core.models import Person
+from dataquality.models import DataQualityIssue, IssueResolverPatch
+
+
+# apply approved patches and create new issues, in case a scraper scrapes a new value
+# which doesn't match with old_value of patch(which can be ignored later if scraping was correct)
+def apply_person_patches(jur_name):
+    count = 0
+    patches = IssueResolverPatch.objects.filter(jurisdiction__name=jur_name,
+                                                status='approved')
+    image_duplicates = []
+    name_duplicates = []
+    for patch in patches:
+        person = Person.objects.get(id=patch.object_id)
```

Comment:
for efficiency would probably make sense to group these by object_id and apply patches together

also, if two patches conflict, we should favor latest, which should be covered here

Reply:
yeah, that sounds like a good idea.

For applying the latest patch, till now I wanted to make changes to the approval view, so I want to mark any other approved resolver to deprecated just at the moment, a new resolver of same category and object_id, get approved, but yes no harm in doing it here.